src/Compilers/CSharp/Test/Emit2/Semantics/RefReadonlyParameterTests.cs

```diff
@@ -40,11 +42,19 @@ private static void VerifyRequiresLocationAttributeSynthesized(ModuleSymbol modu
         }
     }
 
+    private enum VerifyModifiers
+    {
+        None,
+        In,
+        RequiresLocation,
+        Unknown
```

Comment:
>Unknown [](http://example.com/codeflow?start=8&length=7)

When is this supposed to be used.

Reply:
To verify some unusual modifiers. The verification happens in the test, not the common helper method.

Reply:
> To verify some unusual modifiers. The verification happens in the test, not the common helper method.

Consider using name "DoNotVerify" or something similar